### PR TITLE
docs(usr): make usr_05.txt more coherent with Nvim changes

### LINKED ISSUE #22414: docs: Inconsistent instructions on user manual regarding "example vimrc file"
### Problem

In the 5th chapter of the NeoVim user manual, "Set your settings", there's a section called ["The example vimrc file explained,"](https://neovim.io/doc/user/usr_05.html#05.2) which proceeds to explain something which, allegedly, is the "example vimrc file."

This is very strange because, so far in the user manual, no such thing as the "The example vimrc file" was ever introduced.

Moreover, the section continues by saying, "In the first chapter was explained how to create a vimrc file. In this section we will explain the various commands used in this file." That would suggest that the vimrc file created in the first chapter is based on "the example vimrc file." However if we check [that chapter](https://neovim.io/doc/user/usr_01.html#01.2), what we actually create is an empty file. So, there is no example vimrc file, and there is nothing to explain in chapter 5 unless one attempts to explain nothingness itself. But in any case, chapter 5 continues to explain "something," which appears to be a hypothetical example vimrc file, leaving the reader confused.

I believe that this inconsistency comes from a botched modification of Vim's user manual, in which [chapter 1](https://vimhelp.org/usr_01.txt.html#usr_01.txt) indeed creates a vimrc file based on an example file, which is then coherently explained in [chapter 5](https://vimhelp.org/usr_05.txt.html#05.2).

### Expected behavior

Coherent documentation.

### PR DESCRIPTION
**Pull request by lewis6991**
- Make section 05.2 more coherent.
- Remove explanation of options that are already default.

### CODE REVIEW

**Comment by lewis6991** [comment]
@clason, @justinmk: I know this breaks our "runtime files must come from vim" policy, but `usr_05.txt` has already diverged and is quite incoherent.

**Comment by clason** [comment]
We have so far deliberately not added language annotations for help files taken from Vim (in particular the user manual) -- basically, any file that says "VIM [...] MANUAL by Bram" instead of "NVIM REFERENCE MANUAL". I'm not sure we should start now -- but if we do, I'd rather do this fully and in a separate PR; having this only in _some_ files is a bit jarring.

File: runtime/doc/usr_05.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ make Vim start with options set to different values.  Add plugins to extend
 Vim's capabilities.  Or define your own macros.
 
 |05.1|	The vimrc file
-|05.2|	The example vimrc file explained
+|05.2|	Example vimrc commands
```
Comment (clason) [inline runtime/doc/usr_05.txt:13]:
```suggestion
|05.2|	Example vimrc contents
```
as "vimrc commands" are not really a thing? (There's enough confusion of vimscript and ex commands; no reason to add another term to the mix...)

Comment (clason) [reply runtime/doc/usr_05.txt:13]: (My concern was only with the specific term "vimrc commands", which could be misinterpreted as a specific kind of command.)

Comment (lewis6991) [reply runtime/doc/usr_05.txt:13]: > Example commands in a vimrc

?

Comment (clason) [reply runtime/doc/usr_05.txt]: Also good!

**Pull request merged**